DEPS

```diff
@@ -436,6 +436,20 @@ deps = {
      'dep_type': 'cipd',
    },
 
+   'src/third_party/android_tools/google-java-format': {
+     'packages': [
+       {
+        'package': 'flutter/android/google-java-format',
+        'version': 'version:1.7'
+       }
+     ],
+     # We want to be able to format these as part of CI, and the CI step that
+     # checks formatting runs without downloading the rest of the Android build
+     # tooling. Therefore unlike all the other Android-related tools, we want to
+     # download this every time.
+     'dep_type': 'cipd',
+   },
```

Comment:
Can we add a script somewhere to make it easier to create the CIPD package for this?  Or at least instructions?

Reply:
Yes, happy to. Do you have any suggestions, should it be a Wiki page? I have a local `cipd.yaml` on my machine, but I'm not sure where or how to store that information in the repo.

Reply:
I would either make a README.md for it, add a comment to this part of the deps, or add a script in `tools/` that automates it. You probably don't need a cipd.yaml, you could just have the whole cipd invocatoin in a shell script.

Reply:
Actually I have an idea. I'll put the `cipd.yaml` and a `README` in the CIPD package itself, so anybody who goes into the directory to look into it sees what they need to create a new version.

Reply:
Done.